src/components/30-organisms/modal/README.md

```diff
@@ -9,18 +9,25 @@ npm install @axa-ch/modal
 
 ```js
 import '@axa-ch/modal';
-...
-<axa-modal></axa-modal>
+
+// ...
+
+<axa-modal></axa-modal>;
 ```
 
 ## Properties
 
 ### Variant
 
-| Attribute   | Details                   |
-| ----------- | ------------------------- |
-| `open=true` | State of modal visibility |
+| Attribute | Details                                          |
+| --------- | ------------------------------------------------ |
+| `open`    | State of modal visibility (boolean)              |
+| `forced`  | Disables the internal close mechanisms (boolean) |
```

Comment:
My first association with the word "forced" was, that this flag opens up the modal directly. Maybe you could find another word?